pkg/graveler/graveler.go

```diff
@@ -122,6 +122,18 @@ type ResolvedRef struct {
 // MergeStrategy changes from dest or source are automatically overridden in case of a conflict
 type MergeStrategy int
 
+// ConflictResolver is used to resolve conflicts during a merge operation
```

Comment:
Just a suggestion, but I think it's worth mentioning that the resolution is at the object level
```suggestion
// ConflictResolver is used to resolve conflicting objects during a merge operation
```